### PR TITLE
feat(ci): better cli

### PR DESCRIPTION
**Pull request by ShaMan123**
This PR aims to make local dev of fabric easier for the new comer and the seasoned maintainer. So the community can take part easily.
Test suite is quite horrible in terms of DX, this should improve it drastically.
1. Refactored scripts/cmds to be os agnostic (some didn't work on windows). They should all work, please confirm on MacOS
2. Added a debug flag for visual tests to recreate a golden file for the dev to compare changes (this means we need to relay of a [PR I made to qunit](https://github.com/qunitjs/qunit/pull/1681) or simply use my branch)
3. Added a `start` script that handles what's needed to dev via fabricjs.com
4. Consolidated all scripts under one executable

### In Action

https://user-images.githubusercontent.com/34343793/160217304-c2e225d8-7a80-4044-9042-818716d23017.mp4

https://user-images.githubusercontent.com/34343793/160217309-43f0d649-0279-4a48-8047-505a85e8ff74.mp4

### CODE REVIEW

**Comment by asturur** [comment]
going to run those tomorrow morning.
It is important to add PR descriptions Schahar. Imagine you have to sell the PR to someone, every single time.

**Comment by ShaMan123** [comment]
@asturur PR description take 2

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,12 +42,29 @@ Contribute to fabricjs using a fully featured online development environment tha
 
 ### Setting up a local environment
```
Comment (ShaMan123) [inline CONTRIBUTING.md:43]: Should this section be a standalone section or should it reside under the PR section?

**Review by asturur — APPROVED** [review]

**Comment by asturur** [comment]
Thank you for updatind the description, one the question i had was indeed how to run a single test.
Nice CLI!
Merge if you don't want to change anything else

**Comment by asturur** [comment]
Well i ll merge it now because it will help with running test on my pr

**Comment by ShaMan123** [comment]
Excellent.
The only think I'm thinking of is this:
> Added a debug flag for visual tests to recreate a golden file for the dev to compare changes (this means we need to relay of a https://github.com/qunitjs/qunit/pull/1681 or simply use my branch)

I can simply add a env var instead of waiting for quint to allow other flags

**Pull request merged**